packages/web/e2e/uploadTrack.test.ts

```diff
@@ -13,16 +13,52 @@ import {
 } from './page-object-models/upload'
 import { test, waitForUser } from './test'
 import { openCleanBrowser } from './utils'
+import { getAiAttributionUser, getTrack } from './data'
 
-test('should upload a remix, hidden, AI-attributed track', async ({ page }) => {
+test('should upload a track', async ({ page }) => {
+  const trackTitle = `Test track ${Date.now()}`
+  const genre = 'Alternative'
+
+  await page.goto('upload')
+  await waitForUser(page)
+
+  const selectPage = new UploadSelectPage(page)
+  await selectPage.setTracks('track.mp3')
+  await selectPage.continue()
+
+  const editPage = new EditTrackPage(page)
+  await editPage.setArtwork('track-artwork.jpeg')
+  await editPage.setTitle(trackTitle)
+  await editPage.setGenre(genre)
+
+  await editPage.complete()
+
+  const uploadingPage = new UploadFinishPage(page, 'Track')
+  await uploadingPage.assertCompletes()
+
+  // Vist track page
+  await page.getByRole('link', { name: /visit track page/i }).click()
+
+  // Assert title
+  const trackHeading = page.getByRole('heading', {
+    name: trackTitle,
+    level: 1
+  })
+  await expect(trackHeading).toBeVisible()
+})
+
+test.skip('should upload a remix, hidden, AI-attributed track', async ({
```

Comment:
not working? 😢

Reply:
This one I disable to keep the surface area low. There is just a simple track upload test now. But I created a ticket to keep track: https://linear.app/audius/issue/INF-703/re-enable-uploadtracktestts-advanced-test-cases